lib/config/index.js

```diff
@@ -12,4 +30,7 @@ nconf.argv()
     });
 
 nconf.defaults(defaults);
-module.exports = nconf;
\ No newline at end of file
+
+// @TODO: this file gets cached after the first require, we need to redesign how we load the config
+// @TODO: for example: require('ghost-ignition').config()
```

Comment:
But the parent of ignition is always the module which requires ignition. Why is this a bad assumption?
So loading the default/env config files is based on the next parent. So it doesn't matter if you use ignition as a dependency of a dependency or flat in ghost. We just need to solve the cache problem.